unit_tests/mixer_tests.cc

```diff
@@ -17,6 +17,22 @@ TEST(MixerTests, Summation) {
     ASSERT_EQ(result, expected);
 }
 
+// TEST macro violates guidelines
+// NOLINTNEXTLINE(cppcoreguidelines-owning-memory, cppcoreguidelines-avoid-non-const-global-variables)
+TEST(MixerTests, Timeout) {
+    // set up mixer with 2 inputs
+    Pipe<Audio> output{};
+    std::array<Pipe<Audio>, 2> inputs{};
+    {
+        mixer::Mixer<2> mixer(inputs, output);
+
+        // start mixer
+        mixer.run();
+    } // mixer goes out of scope so destructor is called
+
+    ASSERT_TRUE(true); // when run with a timeout this will fail if it doesn't reach this point
```

Comment:
Does this comment mean that if the timeout causes it to timeout or not

Reply:
No, it means that if ctest is run with a timeout it will either succeed and reach that point, returning true, or timeout and fail

Reply:
Ah, that clears it up for me